logstash_async/worker.py

```diff
@@ -198,6 +208,7 @@ def _flush_queued_events(self, force=False):
             else:
                 self._delete_queued_events_from_database()
                 self._reset_flush_counters()
+                self._flush_event.clear()
```

Comment:
Hm, what will happen if user request flush at this moment?

I think nothing, cause we just flushed all messages.

Am I right?

Reply:
It depends, if the new flush request (i.e. flush_event.set()) happens before the clear, nothing happens as it will be cleared immediatly afterwards. If the new flush request is received after the clear, the event will be set and so in the next loop a new flush happens.
Real concurrency on clear and set is not possible as the `threading.Event` class exactly helps for this case by internally managing a lock so only one operation at a time is possible. In other words, the new request can come only before or after the clear operation. Easy for us :).